next-app/src/components/views/Main/Main.style.tsx

```diff
@@ -0,0 +1,61 @@
+import styled from 'styled-components'
+import { getTypographyStyles } from 'styles/fonts'
+import { colors } from 'styles/colors'
+
+export const Wrapper = styled.div`
+  margin: 100px 0;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 100px;
+`
+export const Container = styled.div`
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  /* border: 1px solid red; */
+`
+
+export const ContentsWrapper = styled.div`
+  margin: 0 180px;
+  width: 100%;
+  height: 400px;
+  /* border: 1px solid blue; */
+`
+export const Contents = styled.div`
+  display: flex;
+  height: 100%;
```

Comment:
```suggestion
  height: 400px
  margin: 0 180px;
```
```height: 400px```로 바꾸고 ```margin: 0 180px``` 을 추가하면 Container 와 ContentsWrapper를 사용하지 않아도 똑같은 UI를 구현할 수 있을 것 같아요!

Reply:
해당 코드로 변경 완료했어요 ㅎㅎㅎ